src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -327,7 +324,7 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
 
     var coordinate = new C.EPSG_4326(coordCarto.longitude, coordCarto.latitude, coordCarto.altitude);
 
-    this.scene = Scene(coordinate, viewerDiv, debugMode, gLDebug);
+    this.scene = Scene('EPSG:4978', coordinate, viewerDiv, debugMode, gLDebug);
```

Comment:
Yes, but it was not explicitly stated that the scene respected the 'EPSG:4978', because precisely it was not respected. So, it doesn't hurt to leave a little warning